src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,13 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = {new Truck(), new Bulldozer(),new Excavator()};
+        for (Machine vehicle: machines) {
+            vehicle.doWork();
+            vehicle.stopWork();
+        }
+
```

Comment:
https://github.com/JasonTaranenko/jv-oop/blob/homeWork/checklist.md#please-dont-add-redundant-empty-lines-to-your-code